Makefile

```diff
@@ -54,8 +54,7 @@ BINNAME_D = $(BINNAME_R)_debug
 
 BINDIR = bin
 SRCDIR = src
-OBJDIR = build/obj
-DEPDIR = build/dep
+BLDDIR = build
 
 SUBDIR_R = release
 SUBDIR_D = debug
```

Comment:
If we hardcode the names ```obj``` and ```dep```, shouldn't we also hardcode these? (That is, move these lines below the "Are you sure ..."-header.)

Reply:
Agreed. Since the the variables are irrelevant for the (normal) user experience and a normal user will never have to change them, they can and should be hardcoded.

Reply:
I have hardcoded them.